client/lib/abtest/index.js

```diff
@@ -87,6 +88,16 @@ ABTest.prototype.init = function( name ) {
 		throw new Error( 'A default variation is specified for ' + name + ' but it is not part of the variations' );
 	}
 
+	const languageSlugs = map( config( 'languages' ), 'langSlug' );
+
+	if (
+		testConfig.localeTargets &&
+		! every( testConfig.localeTargets, ( target ) => {
+			return languageSlugs.indexOf( target ) !== -1;
+		} ) ) {
+		throw new Error( 'localeTargets array contains invalid language slugs.' );
```

Comment:
So, this forces a hard crash, ya?  Which is good in this case--because we want the A/B test setter to know there's a problem before the test gets committed?